src/metrics/accuracy.py

```diff
@@ -1,17 +1,18 @@
 import torch
-from typing import Tuple, Union
+from typing import Dict, Tuple, Union
 
 
 # source:
 # https://gist.github.com/weiaicunzai/2a5ae6eac6712c70bde0630f3e76b77b?permalink_comment_id=3662283#gistcomment-3662283
 class TopKAccuracy:
     def __init__(self, k: Union[int, Tuple]):
+        self.k: Tuple
         if isinstance(k, int):
             self.k = (k,)
         else:
-            self.k = k
+            self.k = tuple(k)
```

Comment:
Why do we need tuple(k) here?

Reply:
Almost, mypy didn't pick up that the only way we can get to this branch is for k to be a tuple, so it complained.